src/Driver/Driver.jl

```diff
@@ -223,8 +248,8 @@ function init(; disable_gpu = false, arg_settings = nothing)
     end
 
     # set up the array type appropriately depending on whether we're using GPUs
-    if get(ENV, "CLIMA_GPU", "") != "false" && !Settings.disable_gpu &&
-       CUDAapi.has_cuda_gpu()
+    if get(ENV, "CLIMA_GPU", "") != "false" &&
+       !Settings.disable_gpu && CUDAapi.has_cuda_gpu()
```

Comment:
Do we still need `atyp = CuArrays.CuArray` (in the next line) when `Settings.disable_gpu=false`?

Reply:
If we're _not_ disabling the GPU, then the array type should be `CuArrays.CuArray`.

Reply:
Not important here, but the logic might be easier to follow without double negatives (i.e., change the name to `enable_gpu` and flip the bools)

Reply:
Yes, it confuses me too. But we want `--enable-gpu` to be the default. So the command line option needs to be `--disable-gpu`. I can invert it after parsing though so that it is `Settings.enable_gpu`. I'll do that.